README.md

```diff
@@ -30,38 +30,100 @@ docker login
+
+========================================= Kurtosis Modules =========================================
+GUID   LocalPortBindings
+
+========================================== User Services ==========================================
+GUID   ID   LocalPortBindings
+```
+
+If you pain attention to the screen you will se that we have the enclave but no user service or Kurtosis module has been created yet
+
+Now that we have an enclave, let's put something in it!, go ahead and execute the following command to run a `webserver` (based on the httpd Docker image) inside the enclave:
+
+```
+kurtosis service add demo webserver httpd --ports http=80
+```
+
+And the screen will show us something like this:
+
+```
+Service ID: webserver
+Ports Bindings:
+   http:   80/tcp -> 127.0.0.1:63825
+```
+
+Now, you can go to your browser and check that the webserver is up and running through the local url provided in the http port binding (e.g.: 127.0.0.1:63825)
+
+Let's inspect the enclave again:
+
+```
+kurtosis enclave inspect demo
+```
+
+Now, you should see the webserver listed in the user service slot, like this example:
 
-```javascript
-getServicesResult = await enclaveCtx.getServices()
-services = getServicesResult.value
 ```
+========================================== User Services ==========================================
+GUID                   ID          LocalPortBindings
+webserver-1649186256   webserver   80/tcp -> 127.0.0.1:63825
+```
+
+Finally, if you don't want to continue working with the service you can ger rid of it doing the following:
 
-We haven't started any services yet, so the enclave will be empty. Note how we called `await` on `enclaveCtx.getServices()`. This is because every `enclaveCtx` call is asynchronous and returns a [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise); `await`ing blocks until that value is available.
+```
+kurtosis service rm demo webserver
+```
 
 
 Step Three: Start An Ethereum Network (5 minutes)
 -------------------------------------------------
-Now that we have an enclave, let's put something in it! Ethereum is one of the most popular blockchains in the world, so let's get a private Ethereum network running:
+Now that we know how to create and enclave and add a service, we can go one step ahead and start enjoying one of the most important features Kurtosis have, the Kurtosis modules.
+
+Ethereum is one of the most popular blockchains in the world, so let's get a private Ethereum network running just to executing a Kurtosis module:
 
-```javascript
-loadEthModuleResult = await enclaveCtx.loadModule("eth-module", "kurtosistech/ethereum-kurtosis-module", "{}")
-ethModuleCtx = loadEthModuleResult.value
-executeEthModuleResult = await ethModuleCtx.execute("{}")
-executeEthModuleResultObj = JSON.parse(executeEthModuleResult.value)
-console.log(executeEthModuleResultObj)
+```
+kurtosis module exec 'kurtosistech/ethereum-kurtosis-module'
```

Comment:
Ready for some coolness that I did for the Ava demo? Make this 
```
kurtosis module exec --enclave-id demo 'kurtosistech/ethereum-kurtosis-module'
```
And it'll execute in the already-created `demo` enclave ;)